doc/source/command-line-tools/settings.rst

```diff
@@ -31,10 +33,12 @@ Example Usage
   # Configure settings via bash-style environment variable export
   export IN_TOTO_ARTIFACT_BASE_PATH='/home/user/project'
   export IN_TOTO_ARTIFACT_EXCLUDE_PATTERNS='*.link:.gitignore'
+  export LINK_CMD_EXEC_TIMEOUT='10'
```

Comment:
Have you tried if this works? IIRC it needs a prefix. Would you mind adding a test case for LINK_CMD_EXEC_TIMEOUT to test_user_settings.py?